### DISCUSSION TITLE
fallback: "unstable_blocking" is not classified as ISR  in build

### DISCUSSION THREAD

**Discussion by shunkakinoki** [vercel/next.js#16071] [Help] [answered]
Hi next.js team,
I was using fallback: "unstable_blocking" in the new v9.5.2 and I noticed that the static build was not classified as ISR.
Is this the correct behavior, or is my understanding regarding `fallback: "unstable_blocking"` incorrect?

**Comment by Timer** [comment, marked as answer]
`unstable_blocking` is still considered ISR, correct! Not everything is expected to be working since this is an unstable feature, however.

**Reply by shunkakinoki** [reply]
Thanks for the quick response! (and thanks for creating the issue!)
It's just a feedback of mine, but I've been trying it out on vercel and it's working fantastic for us!
Thank you next.js and vercel, seriously I can't understate how great your work & product is! 🔥🔥🔥
